src/app/organizations/registerOrganization/register-organization.component.html

```diff
@@ -23,6 +23,7 @@ <h1 mat-dialog-title>Add Organization</h1>
       <mat-error *ngIf="name.hasError('required')">Name is required</mat-error>
       <mat-error *ngIf="name.hasError('minlength')">Name should be at least 3 characters</mat-error>
       <mat-error *ngIf="name.hasError('maxlength')">Name should be at most 39 characters</mat-error>
+      <mat-error *ngIf="name.hasError('pattern')">Name can only contain alphanumeric characters and start with an alphabet</mat-error>
     </mat-form-field>
```

Comment:
What is the current behaviour when trying to add two orgs with the same name?
Might be question for @agduncan94

Reply:
The webservice will throw an error